Demo/Demo/FeatureViewControllers/CardDemoViewController.swift

```diff
@@ -98,7 +96,7 @@ class CardDemoViewController: FeatureBaseViewController, UITextFieldDelegate {
     /// Allows us to only enable the pay button once all fields are filled out and we have an order ID
     @objc override func createOrderTapped() {
         createOrder { _ in
-            self.enablePayButton(
+            self.payButton.isEnabled = self.baseViewModel.enablePayButton(
```

Comment:
nitpick: can we call this something other than payButton? It reads too closely to payPalButton to me

Reply:
Totally - does `checkoutButton` make more sense?

Reply:
Totally - updated to `checkoutButton` but if there is an _even better_ name happy to update again